app/models/concerns/hyrax/collection_behavior.rb

```diff
@@ -14,8 +14,15 @@ module CollectionBehavior
     included do
       validates_with HasOneTitleValidator
       self.indexer = Hyrax::CollectionIndexer
+
+      property :collection_type_gid, predicate: ::RDF::Vocab::SCHEMA.additionalType, multiple: false do |index|
+        index.as :symbol, :stored_searchable
+      end
     end
 
+    attr_accessor :collection_type
```

Comment:
Thanks for the responses. This is a great next step, and we'll continue to iterate.